files/preStudies/preStudies.tex

```diff
@@ -0,0 +1,5 @@
+\chapter{Pre-study}
+
+Here follows pre-studies on technologies and tools used throughout this project, conducted in the initial phase of the project.
```

Comment:
and tools to be used throughtout this project

or something?  as written right now it sounds like a prestudie that was conducted after we already used throughout the project?